client/my-sites/plans/plan-overview/style.scss

```diff
@@ -34,8 +34,10 @@
 	}
 	
 	.plan-feature {
-		flex-direction: row;
-		flex-wrap: wrap;
+		@include breakpoint( "<480px" ) {
+			flex-direction: row;
+			flex-wrap: wrap;
+		}
```

Comment:
Could you explain why we set `flex-direction` to `row` in this breakpoint? It should be `column` and I guess it displays like a column because of a side effect of `wrap`.

I'd also like to suggest we simplify the rules in this `.plan-overview` block since right now it's very difficult to reason about what style is applied. The `<480px` breakpoint could (and should) be avoided by introducing specific rules for `.plan-status__info`. Here is what this block could look like if we keep the hack mentioned above:

```css
.plan-overview {
	.plan-feature,
	.plan-status__info {
		align-content: space-between;
		display: flex;
		justify-content: space-between;

		@include breakpoint( ">480px" ) {
			align-items: center;
		}

		&:after {
			display: none;
		}
	}
	
	.plan-feature {
		flex-wrap: wrap;

		.plan-feature__heading {
			margin-right: 10px;
		}
	}

	.plan-status__info {
		flex-direction: column;

		@include breakpoint( ">480px" ) {
			flex-direction: row;
		}
	}
}
```